bundles/org.openhab.binding.panamaxfurman/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This needs to be adapted to the new addon.xml, the PR to adapt the developer docs is not merged yet, so if you need more details, the corresponding issue is here: https://github.com/openhab/openhab-core/issues/2058
But you can also update your branch and look at another binding's addon.xml and adapt it to this initial contribution.